curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/data-cleaning-and-visualizations.english.md

```diff
@@ -14,11 +14,12 @@ videoId: mHjxzFS5_Z0
 
 ```yml
 question:
-  text: Question
+  text: 'When using MatPlotLib's global API, the following code does what in what order?:
```

Comment:
Matplotlib is just capitalized, the apostrophe here in `Matplolib's` needs to be escaped, and the original wording of the question felt awkward. Here's a suggested change

```suggestion
  text: 'When using Matplotlib''s global API, what does the order of numbers mean here?:
```